content/en/apps/tutorials/sms-forms.md

```diff
@@ -61,343 +63,341 @@ To ensure SMS forms function as expected, you will first need to able the follow
 
-To set the validation rules and autoresponses, edit the array corresponding to the `registration` key in `app_settings.json`. Add the following object within the array.
+To set the validation rules and autoresponses, edit the array corresponding to the `registration` key in `app_settings.json`. Add an object within the array as shown below.
 
 ```json
-{
-  "form": "N",
-  "events": [
-    {
-      "name": "on_create",
-      "trigger": "add_patient",
-      "params": "{ \"parent_id\": \"place_id\" }",
-      "bool_expr": ""
-    }
-  ],
-  "validations": {
-    "join_responses": false,
-    "list": [
+"registrations": [
+  {
+    "form": "N",
+    "events": [
       {
-        "property": "place_id",
-        "rule": "regex('^[0-9]{5,13}$')",
-        "translation_key": "messages.validation.household_id"
-      },
-      {
-        "property": "gender",
-        "rule": "iEquals('m') || iEquals('f')",
-        "translation_key": "messages.n.validation.gender"
-      },
+        "name": "on_create",
+        "trigger": "add_patient",
+        "params": "{ \"parent_id\": \"place_id\" }",
+        "bool_expr": ""
+      }
+    ],
+    "validations": {
+      "join_responses": false,
+      "list": [
+        {
+          "property": "place_id",
+          "rule": "regex('^[0-9]{5,13}$')",
+          "translation_key": "messages.validation.household_id"
+        },
+        {
+          "property": "gender",
+          "rule": "iEquals('m') || iEquals('f')",
+          "translation_key": "messages.n.validation.gender"
+        },
+        {
+          "property": "age",
+          "rule": "between(5,130)",
+          "translation_key": "messages.n.validation.age"
+        }
+      ]
+    },
+    "messages": [
       {
-        "property": "age",
-        "rule": "between(5,130)",
-        "translation_key": "messages.n.validation.age"
+        "event": "report_accepted",
+        "translation_key": "messages.n.report_accepted",
+        "recipient": "reporting_unit"
       }
     ]
-  },
-  "messages": [
-    {
-      "event": "report_accepted",
-      "translation_key": "messages.n.report_accepted",
-      "recipient": "reporting_unit"
-    }
-  ]
-}
+  }
+]
 ```
 
-{{% alert title="Note" %}} You can also define your own [validation rules]({{< relref "apps/reference/app-settings#validations" >}}).  {{% /alert %}}
-
-{{% alert title="Note" %}} `translation_key` represents the message that is sent out. This will be defined in a translations file (covered in the translations tutorial). {{% /alert %}}
-
-{{< see-also page="apps/reference/translations" title="Translations" >}}
-
 ### 4. Define a Report Submission form
 
-Add a report submission form by adding the code snippet below to the `forms` object.
+Add a report submission form by adding the form definition key and value(object) pair to the `forms` object as shown below.
```

Comment:
This wording is tricky! It may be because of the multiple use of form... perhaps if you can avoid using the parenthesis it may be help.

Reply:
I've done some rewording. Hopefully, the lines are a bit more clear now.